boards/arm/bcm958402m2_a72/bcm958402m2_a72_defconfig

```diff
@@ -4,6 +4,8 @@
 CONFIG_SOC_SERIES_VIPER=y
 CONFIG_SOC_BCM58402_A72=y
 CONFIG_BOARD_BCM958402M2_A72=y
+CONFIG_ARM64_VA_BITS_36=y
```

Comment:
viper is 36/40 implemented ? can read MMFR on jtag.

Reply:
Maximum valid memory mapped address is `0xF_FFFF_FFFF` (that of normal memory). 36-bits cover all the addresses, based on which I selected 36. I have verified the same- able to access this region.

Btw, I see there are 5 MMFR registers(`ID_MMFR0_EL1` to `ID_MMFR4_EL1`) - which field should be referred? Should I set based on that or this setting (chosen practically) is fine.

Reply:
Chosen practically is better for now.  Its a platform define. 
 If you see no implication (size requirement is met) telling the routine as 36b PA bits is fine I think.

 "implemeID_AA64MMFR0_EL1.PARange" It should tell max support from core. Integration can cap. But dont trust 
  emulation as of now.

Reply:
> Chosen practically is better for now. Its a platform define.
> If you see no implication (size requirement is met) telling the routine as 36b PA bits is fine I think.

This is for MMU page table walk.. Just meeting size requirement- PA_BITS set to 36 bits as that covers entire address range and VA_BITS set to 36 bits as we have 1-to-1 VA-to-PA mapping in place. No other implication.